core/keyserver/keyserver.go

```diff
@@ -30,6 +30,7 @@ import (
 	"google.golang.org/grpc/codes"
 
 	tpb "github.com/google/keytransparency/core/proto/keytransparency_v1_types"
+	spb "github.com/google/keytransparency/impl/proto/keytransparency_v1_service"
```

Comment:
Good point. This will be a problem when importing. We should have monitor APIs in their own service, implement it in `impl`, and implement its core functionality in `core`.